app/assets/javascripts/beak/widgets/ractives/chooser.coffee

```diff
@@ -154,12 +154,18 @@ RactiveChooser = RactiveValueWidget.extend({
 
   widgetType: "chooser"
 
-  observe: {
-    'widget.currentValue': () ->
-      widget        = @get('widget')
-      currentChoice = widget.choices.findIndex( (c) -> c is widget.currentValue )
-      @set('widget.currentChoice', if currentChoice >= 0 then currentChoice else 0)
-      return
+  on: {
+    'chooser-option-change': (event) ->
+      widget       = @get('widget')
```

Comment:
I do prefer the assignments to be whitespace-aligned to the `=`, but it should be for all assignments in this block.  Having a standard 1 space before the `=` is fine, too, as long as it's consistent.